app/controllers/group_health_controller.rb

```diff
@@ -65,10 +66,18 @@ def roles
   end
 
   def groups
-    respond(Group.from("((#{bund}) UNION (#{cantons}) UNION (#{abt_and_below})) " \
+    # geolocations only exist on Group::Abteilung, thus we can't include geolocations in
+    # the queries for other group types
+    abt = Group.from("((#{abteilung})) " \
                        "AS #{Group.quoted_table_name}")
                 .page(params[:page]).per(params[:size] || DEFAULT_PAGE_SIZE)
-                .as_json(only: GROUPS_FIELDS))
+                .as_json(only: GROUPS_FIELDS,
+                         include: { geolocations: { only: GEOLOCATIONS_FIELDS } })
```

Comment:
I first tried to include geolocations in the existing `abt_and_below` query, but this gave me errors in `as_json` with `include: { geolocations: ... }` because the geolocations property only exists for `Group::Abteilung`. If there's a better way to achieve this than concatenation of two queries I'm happy to adapt it.